munimap/application.py

```diff
@@ -16,10 +16,10 @@
 from tempfile import NamedTemporaryFile
 from threading import Lock
 
-from flask import Flask, send_from_directory, request, \
+from flask import Flask, send_from_directory, request as LocalProxyRequet, Request, \
```

Comment:
Minor: Typo in `LocalProxyRequet` -> `LocalProxyRequest` (also accounts for all occurences)